.claude/commands/BMad/tasks/create-brownfield-story.md

```
@@ -0,0 +1,318 @@
+# /create-brownfield-story Task
+
+When this command is used, execute the following task:
+
+<!-- Powered by BMAD™ Core -->
+
+# Create Brownfield Story Task
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document

(MD025, single-title, single-h1)